File: docs/docs/API-Reference/javascript-examples/workflows-api/example-request.ts
import axios from 'axios';

const jobId = 'job_id_1234567890';
const url = `${LANGFLOW_SERVER_URL}/api/v2/workflows`;

const getWorkflowStatus = async () => {
  try {
    const response = await axios.get(url, {
      params: {
        job_id: jobId
      },
      headers: {
        'accept': 'application/json',
        'x-api-key': LANGFLOW_API_KEY
      }
    });
    console.log(response.data);
  } catch (error) {
    console.error('Error getting workflow status:', error);
  }
};

getWorkflowStatus();

File: docs/docs/API-Reference/python-examples/workflows-api/example-request.py
import requests

url = f"{LANGFLOW_SERVER_URL}/api/v2/workflows"
params = {"job_id": "job_id_1234567890"}
headers = {"accept": "application/json", "x-api-key": LANGFLOW_API_KEY}

response = requests.get(url, params=params, headers=headers)
print(response.json())
